### PR TITLE
Fix unbound local error when using the fsoids.py script 

### PR DESCRIPTION
**Pull request by pbauer**
Fix issue mentioned in https://github.com/zopefoundation/ZODB/issues/285#issuecomment-584357646

Closes #329 (duplicate)

### CODE REVIEW

File: src/ZODB/serialize.py [hunk 1]
```diff
@@ -699,7 +699,7 @@ def get_refs(a_pickle):
         if isinstance(reference, tuple):
             oid, klass = reference
         elif isinstance(reference, (bytes, str)):
-            data, klass = reference, None
+            oid, klass = reference, None
```
Comment (jmuchemb) [inline src/ZODB/serialize.py]: Split in 2 lines to avoid the tuple packing/unpacking.

Comment (mgedmin) [reply src/ZODB/serialize.py]: But this way it's nicely symmetric with the case above, where reference is a tuple.

**Comment by mgedmin** [comment]
Python 2.7 fails on Appveyor with syntax errors in mock.

I'm pretty sure this is because Appveyor's default pip is too old to pay attention to python_requires metadata and installs the wrong version of mock.

A `python -m pip install -U pip` step in appveyor.yml ought to fix things.  (Maybe it should be in a separate PR, merged before this one?)

**Comment by mgedmin** [comment]
> A python -m pip install -U pip step in appveyor.yml ought to fix things. (Maybe it should be in a separate PR, merged before this one?)

I should've looked at the appveyor.yml before making assumptions.  We already upgrade pip to the latest version.

mock is actually installed by buildout.  Suggestion: instead of buildout let's use `tox -e py` in the appveyor.yml.

**Comment by ale-rt** [comment]
I would squash and merge the commits if the build passes

**Comment by icemac** [comment]
I restarted the PyPy3 job which had failed. But now it is green. @pbauer Feel free to merge.

**Comment by mgedmin** [comment]
Oops my finger slipped and I accidentally did the suggested squash-and-merge!

Thank you everyone!

**Pull request merged**